OrbitLinuxTracing/UprobesUnwindingVisitorTest.cpp

```diff
@@ -112,6 +200,34 @@ TEST(UprobesCallstackManager, OneUprobe) {
                   TestCallstackToStringPairVector(expected_cs)));
 }
 
+TEST(UprobesCallstackManager, OneUprobeNoCallstackOnUretprobe) {
+  constexpr pid_t tid = 42;
+  std::vector<unwindstack::FrameData> unwound_cs, expected_cs, processed_cs;
+  UprobesCallstackManager callstack_manager{};
+  // Begin FUNCTION().
```

Comment:
I don't understand what this comment here means. Can you please add more explanation here?

Reply:
It means that we simulate the call of the function FUNCTION. I'll change the comments.